katago_server/runs/viewsets/run.py

```diff
@@ -31,3 +31,14 @@ def current(self, request):
             return Response({'error': 'No active run.'}, status=status.HTTP_404_NOT_FOUND)
         self.kwargs["pk"] = current_run.pk
         return self.retrieve(request)
+
+    @action(detail=False, methods=["GET"])
+    def current_for_client(self, request):
+        """
+        API endpoint that gives only the fields of a run that self-play clients need.
+        :return:
+        """
+        current_run = Run.objects.select_current()
+        if current_run is None:
+            return Response({'error': 'No active run.'}, status=status.HTTP_404_NOT_FOUND)
+        return Response(RunSerializerForClient(current_run,context={'request': request}).data)
```

Comment:
Extract `run_serilaizer = RunSerializerForClient(current_run,context={'request': request})` for better readability.

Reply:
ok